ntp/4.2.8p15/README.md

```diff
@@ -0,0 +1,15 @@
+## Build Instructions
+
+### Build wolfSSL
++ Configure wolfSSL with `./configure --enable-certgen --enable-dsa --enable-cmac --enable-opensslextra --enable-opensslall --enable-keygen`. Add `--enable-debug` if you want to enable the debug version of wolfSSL.
++ Compile with `make`.
++ Install wolfSSL into /usr/local with `sudo make install`.
+
+### Build NTP
++ Download ntp 4.2.8p15 with `curl -O https://www.ntp.org/downloads/ntp-4.2.8p15.tar.gz`.
```

Comment:
Appears that ntp.org is down at the moment for me ("Not Found" error when visiting site).  I grabbed this tar.gz from elsewhere and verified these steps work though.  Guessing ntp.org will come back soon.

Reply:
Looks to be back up now, at least for me.